core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -73,6 +73,10 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
   // The number of pending tasks which is locality required
   protected var localityAwareTasks = 0
 
+  // Executors that have been marked as disabled by the cluster manager, and are pending
```

Comment:
Also can you change this comment to "Executors that have been removed by the cluster manager, so no tasks should be scheduled on them, but we're still waiting to learn by they were removed"? As-is, it's hard for a reader to quickly understand how this is different from executorsPendingToRemove